mass_mailing_partner/views/mail_mass_mailing_contact_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<record model="ir.ui.view" id="view_mail_mass_mailing_contact_tree">
+    <field name="name">mail.mass_mailing.contact.tree.partner</field>
+    <field name="model">mail.mass_mailing.contact</field>
+    <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_contact_tree"/>
+    <field name="arch" type="xml">
+        <field name="email" position="after">
+            <field name="partner_mandatory" invisible="1"/>
+            <field name="partner_id" attrs="{'required': [('partner_mandatory', '=', True)]}"/>
```

Comment:
```
attrs="{'required': [('partner_mandatory', '=', True)]}"
```
is not neccesary, because if ```partner_mandatory``` is True. If no partner is filled then addon will search (or create) one with that email, check for duplicates and assigne partner field.

But if attrs defined, user can not create a contact filling only email. User is expecting that addon search for the partner (or create a new partner with that email if not exists)